### PR TITLE
[TIMOB-24397] iOS: WebView - load a webpage with request headers

### PR DESCRIPTION
**Pull request by chreck**
Jira: https://jira.appcelerator.org/browse/TIMOB-24397

Move the setting of the request header into the function loadURLRequest.

### CODE REVIEW

**Review by hansemannn — CHANGES_REQUESTED** [review]

**Review by hansemannn — APPROVED** [review]

File: iphone/Classes/TiUIWebView.m [hunk 1]
```diff
@@ -196,11 +196,24 @@ - (id)accessibilityElement
...
+
+    NSDictionary *requestHeaders = [[self proxy] valueForKey:@"requestHeaders"];
+    if(requestHeaders!=nil)
+    {
+        // set the new headers
+        for(NSString *key in [requestHeaders allKeys]){
+            NSString *value = [requestHeaders objectForKey:key];
+            //NSLog(@"[INFO] loadURLRequest header key %@ value %@", key, value);
+            [request setValue:value forHTTPHeaderField:key];
+        }
```
Comment (hansemannn) [inline iphone/Classes/TiUIWebView.m]: I've done it for you now.

**Comment by hansemannn** [comment]
CR approved!

**Comment by longton95** [comment]
FR passed.
Webview now loads webpages with request headers tested using the test case in the ticket TIMOB-24397 as well as the test cases in TIMOB-17467 and TIMOB-24329


ENV
```
SDK Version : Local 6.1.0 
Mac OS Version : 10.12.2
Xcode Version : Xcode 8.2 Build version 8C38
Appc CLI : 6.1.0
Appc NPM : 4.2.9-1
Node : v4.6.0
Device: iPhone 6 (10.2.1) iPhone 5 (9.3.5)
```

**Review by longton95 — APPROVED** [review]
FR passed.
Webview now loads webpages with request headers tested using the test case in the ticket TIMOB-24397 as well as the test cases in TIMOB-17467 and TIMOB-24329


ENV
```
SDK Version : Local 6.1.0 
Mac OS Version : 10.12.2
Xcode Version : Xcode 8.2 Build version 8C38
Appc CLI : 6.1.0
Appc NPM : 4.2.9-1
Node : v4.6.0
Device: iPhone 6 (10.2.1) iPhone 5 (9.3.5)
```

**Pull request merged**